server/src/main/java/io/deephaven/server/session/TicketRouter.java

```diff
@@ -74,6 +74,9 @@ public TicketRouter(
                             + resolver.ticketRoute());
                 }
             }
+            if (resolver instanceof WantsTicketRouter) {
+                ((WantsTicketRouter) resolver).setTicketRouter(this);
+            }
```

Comment:
Not that I would expect implementations to do meaningful work in `setTicketRouter`, but I would want this to be done last during construction (ie, after `byteResolverMap`, et al, is fully populated).